flutter/lib/src/sentry_flutter.dart

```diff
@@ -27,19 +22,17 @@ mixin SentryFlutter {
     FlutterOptionsConfiguration optionsConfiguration, {
     AppRunner? appRunner,
     PackageLoader packageLoader = _loadPackageInfo,
-    iOSPlatformChecker isIOSChecker = isIOS,
-    AndroidPlatformChecker isAndroidChecker = isAndroid,
     MethodChannel channel = _channel,
+    SentryFlutterOptions? options,
```

Comment:
is this needed because of testing?

Reply:
Yes, I removed the iOS and Android platform checkers in favor of  `options.platformChecker.platform`. I mean we could add more individual checks for the rest of the platforms but I don't think that's the way to go.

It's outlined in https://github.com/getsentry/sentry-dart/projects/1#card-50187067

Reply:
Instead of `SentryFlutterOptions` we also could just expose the `PlatformChecker`. I'm not really sure what I prefer though.

Reply:
looks like PlatformChecker would be better then as people would not use the `options` to hack stuff around unless you need it for testing more things